core/src/main/scala/cell/HandlerPool.scala

```diff
@@ -205,6 +250,31 @@ class HandlerPool(parallelism: Int = 8, unhandledExceptionHandler: Throwable =>
     })
   }
 
+  /**
+   * If a cell is triggered, it's `init` method is
+   * run to both get an initial (or possibly final) value
+   * and to set up dependencies. All dependees automatically
+   * get triggered.
+   *
+   * @param cell The cell that is triggered.
+   */
+  def triggerExecution[K <: Key[V], V](cell: Cell[K, V]): Unit = {
+    if (cell.markAsRunning())
+      execute(() => {
+        val completer = cell.asInstanceOf[CellImpl[K, V]]
```

Comment:
This relies on the implementation of Cell/CellCompleter. Unfortunately, I am not able to change the method to take a CellCompleter as long as there is no way to get the completer of a cell. (This would be required in whenNext.) Is it OK to add a
`private[cell] def completer: CellCompleter`
to trait `Cell`?